### ISSUE TITLE
Headings and links in a11y module should be unique

### ISSUE THREAD

**Issue by oysteinmoseng** [highcharts/highcharts#10541]
Top level headings and all links should include chart title.

This is important when multiple charts are presented on a page.

**Comment by oysteinmoseng** [comment]
Related: #11348.
